profile/README.md

```diff
@@ -7,6 +7,8 @@
 
 **Our Mission** – We empower entrepreneurs by growing their business and maximizing their income. Our complete set of automated appointment scheduling, payment processing, same-business-day transfer, client management, and built-in marketing solutions was designed to empower professionals to focus on being creators, not admins. We believe technology can help our members reach their full potential of entrepreneurship.
 
+**Our Stacks** – We like to keep things simple. On the backend, we currently use Kotlin and Typescript for our new services, our legacy system is written in Ruby. Four our web and mobile clients, we use Typescript and React (native).
+
```

Comment:
thoughts:
- I'm not sure if we want to use `legacy` here, may give the wrong impression.
- I thought back & forth on whether or not it's an issue to say what languages (and by extension technologies) we use. I lean towards it's not a unique issue here. We do tell candidates what technologies we use in both our job listings and verbally before any NDAs, and so it's already public information elsewhere.